plugin/storage/integration/es_index_cleaner_test.go

```diff
@@ -32,7 +32,7 @@ const (
 	samplingIndexName     = "jaeger-sampling-2019-01-01"
 	spanIndexName         = "jaeger-span-2019-01-01"
 	serviceIndexName      = "jaeger-service-2019-01-01"
-	indexCleanerImage     = "jaegertracing/jaeger-es-index-cleaner:latest"
+	indexCleanerImage     = "jaegertracing/jaeger-es-index-cleaner:local-test"
 	rolloverImage         = "jaegertracing/jaeger-es-rollover:1.57.0"
```

Comment:
Same issue

Reply:
The jaeger-es-index-cleaner is now pulling the local image, and there are no longer any log messages for this process.

Reply:
Rollover image is also pulled by published version instead of being built locally

Reply:
https://github.com/jaegertracing/jaeger/pull/5714#issuecomment-2212419954
so is the current implementation for  es-index-cleaner is good enough

Reply:
i will make appropriate changes